async/src/traits/ring_buffer.rs

```diff
@@ -0,0 +1,8 @@
+use crate::consumer::AsyncConsumer;
+use crate::producer::AsyncProducer;
+use ringbuf::traits::RingBuffer;
+
+pub trait AsyncRingBuffer: RingBuffer + AsyncProducer + AsyncConsumer {
+    fn wake_producer(&self);
+    fn wake_consumer(&self);
+}
```

Comment:
Add two methods for waking producer / consumer for trait AsyncRingBuffer

Reply:
I doubt that such low-level controls should be exposed to user but I can't suggest a better way to do this yet.

Reply:
Yes. Just realized it's interesting that Rust does not actually have private trait fn.